test/python/expected/sql_validation_framework/expected_drop.out

```diff
@@ -47,6 +48,7 @@ Unexpected drop found for procedure sys.babelfish_update_collation_to_default in
 Unexpected drop found for procedure sys.babelfish_update_collation_to_default in file babelfishpg_tsql--2.4.0--3.0.0.sql
 Unexpected drop found for procedure sys.babelfish_update_collation_to_default in file babelfishpg_tsql--2.5.0--3.0.0.sql
 Unexpected drop found for procedure sys.babelfish_update_collation_to_default in file babelfishpg_tsql--2.6.0--3.0.0.sql
+Unexpected drop found for procedure sys.babelfish_update_collation_to_default in file babelfishpg_tsql--2.7.0--3.0.0.sql
```

Comment:
Do we need this function and procedure?

Reply:
As @Deepesh125 suggested in earlier comment that it is not required. Removed it.